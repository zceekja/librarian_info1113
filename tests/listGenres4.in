add book book1.csv 111111
ADD BOOK book1.csv 111112
add book book1.csv 111118
list genres