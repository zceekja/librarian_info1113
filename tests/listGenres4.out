user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Hobbit (J.R.R. Tolkien).

user: Successfully added: Harry Potter 1 (J.K. Rowling).

user: Fantasy
Historical Fiction

user: 